packages/grafana-ui/src/components/PieChart/PieChartWithLegend.tsx

```diff
@@ -1,8 +1,30 @@
 import React, { FC } from 'react';
-import { Props as PieChartProps } from './PieChart';
+import { VizLayout } from '../VizLayout/VizLayout';
+import { LegendDisplayMode, VizLegendItem } from '../VizLegend/types';
+import { VizLegend } from '../VizLegend/VizLegend';
+import { PieChart, DEFAULT_COLOR, Props as PieChartProps } from './PieChart';
 
 export interface Props extends PieChartProps {}
 
-export const PieChartWithLegend: FC<Props> = ({ width, height, ...restProps }) => {
-  return <div>Need VizLayout in grafana/ui</div>;
+export const PieChartWithLegend: FC<Props> = ({ values, width, height, ...restProps }) => {
+  const legendItems = values.map<VizLegendItem>((value) => {
+    console.log(value.title, value.color);
```

Comment:
This should probably be removed.

Reply:
Oh, dammit. Even greped the diff for console logs to get rid of them all, but forgot to save this file after the change. Thanks.